src/pages/AddLiquidity/index.tsx

```diff
@@ -150,6 +153,16 @@ export default function AddLiquidity({
 
   useEffect(() => setShowCapitalEfficiencyWarning(false), [baseCurrency, quoteCurrency, feeAmount])
 
+  useEffect(() => {
+    if (typeof parsedQs.minPrice === 'string' && parsedQs.minPrice !== leftRangeTypedValue) {
+      onLeftRangeInput(parsedQs.minPrice)
+    }
+
+    if (typeof parsedQs.maxPrice === 'string' && parsedQs.minPrice !== rightRangeTypedValue) {
```

Comment:
i believe you want `parsedQs.maxPrice` here?

Reply:
Good catch!